app/controllers/vacancies_controller.rb

```diff
@@ -1,6 +1,10 @@
 class VacanciesController < ApplicationController
   helper_method :job_application
 
+  attr_accessor :canonical_href
```

Comment:
It doesn't quite make sense to have an attr_accessor on a controller. `canonical` could be a helper method, and I think it might actually be better off in `VacanciesHelper`.